include/knowrob/ros/ROSInterface.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, Sascha Jongebloed
+ * All rights reserved.
+ *
+ * This file is part of KnowRob, please consult
+ * https://github.com/knowrob/knowrob for license details.
+ */
+
+
+#ifndef KNOWROB_ROSINTERFACE_H
+#define KNOWROB_ROSINTERFACE_H
+
+#include "knowrob/ros/ROSInterface.h"
+// KnowRob
+#include "knowrob/knowrob.h"
+#include "knowrob/Logger.h"
+#include "knowrob/KnowledgeBase.h"
+#include "knowrob/queries/QueryParser.h"
+#include "knowrob/formulas/ModalFormula.h"
+// ROS
+#include <ros/ros.h>
+#include <ros/console.h>
+#include <ros/package.h>
+#include <iai_knowledge_msgs/GraphAnswer.h>
```

Comment:
Why "iai_knowledge_msgs"? Could we use `<knowrob/ros/msgs>` instead? External people might not associate "iai" with knowrob, so I suggest to put knowrob into the name

Reply:
I thought this was the standard we used with all software packages of our institute, e.g. like the json_prolog_msgs. But I should add the dependencie atleast to the rosinstall to make it easier to install.